packages/web/package.json

```diff
@@ -65,6 +66,7 @@
     "jest-environment-jsdom": "28.1.3",
     "postcss": "8.4.16",
     "postcss-cli": "10.0.0",
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
@adamkudrna This is the `devDependency` and polyfill for `js-dom`, eg. for tests to work.

Reply:
It is polyfill only for test cases!
This is not from my head. Since I kept getting errors about this while writing these tests, I found a solution on stackoverflow and implemented it after discussing with @literat